test/contract/specs/list-animal.test.js

```diff
@@ -0,0 +1,56 @@
+const {Pact, Matchers} = require('@pact-foundation/pact');
```

Comment:
use a better name for the file, I like the `{service}.contract.test.js` convention